src/backend/codegen/include/codegen/utils/utility.h

```diff
@@ -32,7 +33,7 @@ namespace gpcodegen {
  * @return A pointer to the specified argument, or NULL if the specified
  *         position was beyond the end of function's arguments.
  **/
-llvm::Argument* ArgumentByPosition(llvm::Function* function,
+static llvm::Argument* ArgumentByPosition(llvm::Function* function,
```

Comment:
Why did you make this a static function? Won't this be accessed from our code generators?